BackofficeBundle/Security/Authorization/Voter/GroupSiteVoter.php

```diff
@@ -76,6 +77,9 @@ public function vote(TokenInterface $token, $object, array $attributes)
                 }
             }
         }
+        if ($user instanceof UserInterface && $user->isSuperAdmin()) {
```

Comment:
i think than you can do this check before the other  ``ìf (($user = $token->getUser()) instanceof GroupableInterface)``